menon2021re v1

kevins99 opened the pull request:
Hello, please find an initial version of the Camera ready paper. No errors on our end while formatting, just a few warnings. 
Thank you!

koustuvsinha commented on menon2021re/journal/metadata.yaml:
```diff
@@ -55,8 +40,8 @@ keywords: rescience c, rescience x
-  - doi: 
+  - url: https:/github.com/alchemi5t/MNADrc
+  - doi: 10.5281/zenodo.4681515
   - swh: 
```
Please add the Software Heritage Foundation URI for your code (instructions in README)

koustuvsinha commented on menon2021re/journal/metadata.yaml:
```diff
@@ -66,22 +51,33 @@ data:
-           some interviews and we published an article introducing ReScience in
-           PeerJ. Based on our experience at managing the journal during these
-           three years, we think the time is ripe for proposing some changes."
+abstract: "Scope of Reproducibility
```
You can probably remove the line breaks from the abstract here.

kevins99 commented:
HI!
Have made the following changes:
1) Added SWH in metadata.yaml
2) Removed line breaks from abstract in metadata.yaml
Thank you!

koustuvsinha commented:
Compilation works on my end. However, I noticed you have removed the symlinks and added the folders from template directly. Please revert that, as the symlinks help to centralize any update to the main template in one place. From your journal folder, clean up the PR using the following:

```
rm -rf Makefile article.py header.tex rescience.cls roboto source-code-pro source-sans-pro source-serif-pro yaml-to-bibtex.py yaml-to-latex.py yaml-to-markdown.py metadata.tex
```

And then add the symlinks:

```
    ln -s ../../template/Makefile .
    ln -s ../../template/article.py .
    ln -s ../../template/header.tex .
    ln -s ../../template/rescience.cls .
    ln -s ../../template/roboto .
    ln -s ../../template/source-code-pro .
    ln -s ../../template/source-sans-pro .
    ln -s ../../template/source-serif-pro .
    ln -s ../../template/yaml-to-bibtex.py .
    ln -s ../../template/yaml-to-latex.py .
    ln -s ../../template/yaml-to-markdown.py .
```

kevins99 commented:
We have incorporated the suggested changes regarding symlinks. 
However, post adding the symlinks and deleting the required folders, we onoticed that our names were bot being displayed. 
We fixed this by setting the abstract in metadata.yaml to " "(Empty string). We hope that will not be an issue and apologise for the delay.

koustuvsinha commented:
Looks good to me. I also reviewed the pdf content, looks good.
